src/css/hero.css

```diff
@@ -0,0 +1,186 @@
+.hero {
+  padding-bottom: 80px;
+}
+.hero .hero-container {
+  min-height: 709px;
+  padding: 20px;
+  padding-top: 425px;
+  padding-bottom: 14px;
+
+  border-radius: 16px;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  background-image: linear-gradient(
+      357deg,
+      rgba(0, 0, 0, 0.4) 39.23%,
+      rgba(0, 0, 0, 0) 66.28%
+    ),
+    url('../img/hero/bg-mobile.jpg');
+}
+@media only screen and (min-device-pixel-ratio: 2),
+  (min-resolution: 192dpi),
+  (min-resolution: 2dppx) {
+  .hero .hero-container {
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-mobile@2x.jpg');
+  }
+}
+@media only screen and (min-width: 768px) {
+  .hero .hero-container {
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-tablet.jpg');
+  }
+}
+@media only screen and (min-width: 768px) and (min-resolution: 192dpi),
+  (min-width: 768px) and (min-device-pixel-ratio: 2),
+  (min-width: 768px) and (min-resolution: 2dppx) {
+  .hero .hero-container {
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-tablet@2x.jpg');
+  }
+}
+@media only screen and (min-width: 1280px) {
+  .hero .hero-container {
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-desktop.jpg');
+  }
+}
+@media only screen and (min-width: 1280px) and (min-resolution: 192dpi),
+  (min-width: 1280px) and (min-device-pixel-ratio: 2),
+  (min-width: 1280px) and (min-resolution: 2dppx) {
+  .hero .hero-container {
+    background-image: linear-gradient(
+        357deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
+      url('../img/hero/bg-desktop@2x.jpg');
+  }
+}
+
+.hero-header {
+  max-width: 295px;
+  margin-bottom: 40px;
+
+  color: var(--white);
+  font-size: 38px;
+  font-weight: 700;
+  line-height: 1.05;
+}
+.hero-header-english {
+  color: var(--soft-green);
+}
+
+.hero-buttons-wrapper {
```

Comment:
`.hero-buttons-wrapper` is not needed here
scroll-to-bottom button should be absolute positioned with `bottom` and `right` values